lib/adaptors/serialport.js

```diff
@@ -119,3 +119,7 @@ Adaptor.prototype.onRead = function read(callback) {
 Adaptor.prototype.close = function close(callback) {
   this.serialport.close(callback);
 };
+
+Adaptor.prototype.newSerialport = function() {
+  return new SerialPort(this.conn, {});
+};
```

Comment:
@stewart I can't make that test pass as you suggested, if you want to try to fix it as suggested you are welcome. The only way I could make them past is as they are right now.